### PR TITLE
Adding requirements txt file to cudf docs directory

### PR DESCRIPTION
**Pull request by rajkaramchedu-nvidia**
Adding the missing ``requirements.txt`` file to ensure that the local docs (static html) can be built. Without this file it is not possible to build doc html files on a local machine. I am requesting this to be merged into the ``master`` branch. This is my first PR in this repo, and if my PR is missing any info, please let me know, or else, bless it! :- )

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by rajkaramchedu-nvidia** [comment]
BTW, I don't seem to have permissions to add labels?

**Comment by rajkaramchedu-nvidia** [comment]
Just retargeted to ``branch-0.9`` (Edit and pick the branch from the ``base`` drop down). Wonder why it suddenly added an extra file ``cpp/CMAkeLists.txt`` to my PR? I didn't commit that.

File: cpp/CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #=============================================================================
 cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
 
-project(CUDA_DATAFRAME VERSION 0.7.2 LANGUAGES C CXX CUDA)
+project(CUDA_DATAFRAME VERSION 0.8.0 LANGUAGES C CXX CUDA)
```
Comment (randerzander) [inline cpp/CMakeLists.txt]: Not sure why branch-0.9's CMakeLists.txt has cudf version as 0.7.2.

I'd think it should be 0.9.0 now. @raydouglass can you confirm?

**Comment by rajkaramchedu-nvidia** [comment]
I thought too that it was odd to see "requirement.txt" instead of "requirements.txt". Note that the existing README page already has the instructions to use "requirement.txt" but the txt file is missing. All I did in this PR is to add the missing "requirement.txt" file. I guess now my thinking is, if pip install step should not be used, referring instead to the conda environment (which I agree makes better sense), then what would be the correct instructions in this README file?

**Comment by randerzander** [comment]
@rajkaramchedu-nvidia if you and @kkraus14 agree, I think we can eliminate the requirement.txt reference from the the docs readme, and point people to the cudf dev conda environment instead of having them install anything through pip.

**Comment by rajkaramchedu-nvidia** [comment]
I agree. So, with that, i.e., removing the requirements.txt reference from this README, I will need to put in the exact commands a develoer-user must execute to build the docs locally. My opinion is we should provide a "copy/paste"-easy level instructions for anyone who wants to build the docs. What I mean is, simply adding a reference to https://github.com/rapidsai/cudf/blob/branch-0.9/conda/environments/cudf_dev_cuda10.0.yml here wouldn't be of much help. We have to say what the user should do, i.e., what commands she should execute to install these dependencies, or make sure these dependencies are fulfilled. For example, should we say something like "you must install conda environment first with these dependencies, and only then come here to this cudf docs directory and execute make html"? Is that your thinking, @randerzander?

**Comment by rajkaramchedu-nvidia** [comment]
I see. Makes sense. I will get to it soon.

**Comment by rajkaramchedu-nvidia** [comment]
I think we should probably close this PR without merging.

**Pull request closed without merging**